tests/Agent/UnitTests/Core.UnitTest/AgentHealth/AgentHealthReporterTests.cs

```diff
@@ -42,24 +44,24 @@ public void SetUp(bool enableLogging = true)
         [TearDown]
         public void TearDown()
         {
+            _enableLogging = true;
```

Comment:
It might be clearer if you moved this line into the SetUp method, and if you do that you don't need to use the field initializer to set an initial value either.

Reply:
Fair enough.